docs/api/app.md

```diff
@@ -904,6 +904,15 @@ Returns [`ProcessMetric[]`](structures/process-metric.md): Array of `ProcessMetr
 
 Returns [`GPUFeatureStatus`](structures/gpu-feature-status.md) - The Graphics Feature Status from `chrome://gpu/`.
 
+### `app.getGPUInfo([callback])`
+
+Returns `Object` - GPU Information available in chromium's GPUInfo object. It's contains the version and driver information that's shown on `chrome://gpu` page.
```

Comment:
Returning an object and accepting a callback 😕 

Would it be better to callback with everything, it seems really weird for a single API to be both synchronous and asynchronous

Reply:
Yeah, it definitely is hacky. I'm leaning towards the promise way now. Will make changes and update the branch. Thanks @MarshallOfSound and @YurySolovyov !